app/scripts/Annotations1dTrack.js

```diff
@@ -3,21 +3,28 @@ import * as PIXI from 'pixi.js';
 
 import PixiTrack from './PixiTrack';
 
+import pubSub from './services/pub-sub';
+
+// Maximum delay in ms between mousedown and mouseup that is registered as a
+// click. Note we need to use mousedown and mouseup as PIXI doesn't recognize
+// click events with out current setup. Since most UIs treat long clicks as
+// either something special or a cancelation we follow best practices and
```

Comment:
Do you understand what it is that prevents click events from being registered? ... Sounds like it's something we do, and not something intrinsic to Pixi?

Reply:
Yes I know what it is and yes it's something on our side. If you want hover you need to constantly flush the PIXI object. That's fine for gaming but for a web app it's bad because it drains the battery but using too much CPU even if nothing changes. Why are you asking?

Reply:
Thanks: The context helps me. That was what I wanted to know... It's something we could turn on, but we have good reasons not to. In the original context it sounded like it might be a bug we didn't understand, or something else.

Reply:
If you're curious about the details, take a look here: https://stackoverflow.com/questions/27025843/interactive-graphics-with-pixijs-mouseover-doesnt-fire-while-click-does
The expensive thing (which is not a problem for games but for us) is looping updater which leads to a constant CPU usage of ~10-15% even when nothing changes. Please don't ask me why click events work for them but not us... I assume it's because of our event forwarding system but I could be wrong. 😅 I am just happy the simulated click works